nixos/modules/virtualisation/openvswitch.nix

```
@@ -129,6 +129,8 @@ in {
         PIDFile = "/run/openvswitch/ovs-vswitchd.pid";
         # Use service type 'forking' to correctly determine when vswitchd is ready.
         Type = "forking";
+        Restart = "always";
```

Would `on-failure` also work here?

```suggestion
        Restart = "on-failure";
```